lib/pg_query/deparse.rb

```diff
@@ -338,9 +339,11 @@ def deparse_insert_into(node)
       output = ['INSERT INTO']
       output << deparse_item(node['relation'])
 
-      output << '(' + node['cols'].map do |column|
-        deparse_item(column)
-      end.join(', ') + ')'
+      if node['cols']
```

Comment:
:+1: Just verified this in the Postgres source, see https://github.com/postgres/postgres/blob/master/src/include/nodes/parsenodes.h#L1209 :)